src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/RunManifests.scala

```diff
@@ -1,242 +0,0 @@
-/*
- * Copyright (c) 2016-2019 Snowplow Analytics Ltd. All rights reserved.
- *
- * This program is licensed to you under the Apache License Version 2.0,
- * and you may not use this file except in compliance with the Apache License Version 2.0.
- * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the Apache License Version 2.0 is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
- */
-package com.snowplowanalytics.snowplow.analytics.scalasdk
-
-// Scala
-import scala.collection.convert.decorateAsJava._
-import scala.collection.convert.decorateAsScala._
-
-// AWS DynamoDB
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
-import com.amazonaws.services.dynamodbv2.model._
-import com.amazonaws.services.dynamodbv2.document.Table
-
-// AWS S3
-import com.amazonaws.services.s3.AmazonS3
-import com.amazonaws.services.s3.model.{ ListObjectsV2Request, ListObjectsV2Result }
-
-/**
-  * Wrapper class for run manifests table
-  * Can be used instead of functions from `RunManifests` module
-  *
-  * @param dynamodb AWS DynamoDB client
-  * @param tableName existing DynamoDB table name with run manifests
-  */
-class RunManifests(dynamodb: AmazonDynamoDB, tableName: String) {
```

Comment:
I was thinking more about https://www.scala-lang.org/api/current/scala/deprecated.html rather than deleting it compltely. It just unties our hands about deleting it in 1.x series, i.e. its not something users should rely on anyways.

Reply:
So, should I make it `deprecated` or leave it as removed ?

Reply:
`deprecated`